docs/operator-manual/core.md

```diff
@@ -0,0 +1,95 @@
+# Argo CD Core
+
+## Introduction
+
+Argo CD Core is a different installation that runs Argo CD in headless
+mode. With this installation, you will have a fully functional GitOps
+engine capable of getting the desired state from Git repositories and
+applying it in Kubernetes.
+
+The following groups of features won't be available in this
+installation:
+
+- Argo CD RBAC model
+- Argo CD API
+- OIDC based authentication
+- Argo CD web application
+- Argo CD CLI
+- Multi-tenancy
+
+A few use-cases that justify running Argo CD core are:
+
+- As a cluster admin, I want to rely on Kubernetes RBAC only.
+- As a devops engineer, I don't want to learn a new API or depend on
+  another CLI to automate my deployments. I want instead rely in
+  Kubernetes API only.
+- As a cluster admin, I don't want to provide Argo CD UI or Argo CD
+  CLI to developers.
+
+## Architecture
+
+Because Argo CD is designed with a component based architecture in
+mind, it is possible to have a more minimalist installation. In this
+case fewer components are installed and yet the main GitOps
+functionality remains operational.
+
+In the diagram below, the Core box, shows the components that will be
+installed while opting for Argo CD Core:
+
+![Argo CD Core](../assets/argocd-core-components.png)
+
+Note that even if the Argo CD controller can run without Redis, it
+isn't recommended. The Argo CD controller uses Redis as an important
+caching mechanism reducing the load on Kube API and in Git. For this
+reason, Redis is also included in this installation method.
+
+## Installing
+
+Argo CD Core can be installed by applying a single manifest file that
+contains all the required resources.
+
+Example:
+
+```
+export ARGOCD_VERSION=<desired argo cd release version>
```

Comment:
Maybe note that the version should start with `v` (provide an example).

Reply:
added